wire/jay_input.txt

```diff
@@ -151,6 +161,9 @@ event input_device {
     left_handed: u32,
     natural_scrolling_available: u32,
     natural_scrolling_enabled: u32,
+    click_method: i32,
+    middle_button_emulation_available: u32,
+    middle_button_emulation_enabled: u32,
```

Comment:
Existing events cannot be modified.

Reply:
hmm, how should adding these work then?

Reply:
Same as `CalibrationMatrix`.

Reply:
Any other preferences?

I'm thinking there might be other fields like this added in the future, so it probably makes more sense to create separate events for each setting. What do you think?

Reply:
Fine with me.